install/blazor-server/install.yml

```diff
@@ -0,0 +1,16 @@
+id: blazor-server
+
+name: Blazor Server
+
+title: Blazor Server
+description: Instrument and monitor your Blazor Server application with our .NET agent. Analyze your performance data right out of the box. 
```

Comment:
"Instrument and monitor your Blazor Server application with our .NET APM agent."